src/ripple/app/main/Application.cpp

```diff
@@ -281,7 +281,7 @@ class ApplicationImp
             auto const ms (ceil <std::chrono::milliseconds> (elapsed));
 
             {
-                std::unique_lock <std::mutex> lock (m_mutex);
```

Comment:
Since we're messing with this, aren't we better off making `m_lastSample` an `std::atomic`? Then lines 283-286 can be reduced to just:

    lastSample_ = ms;

Similarly, the `get` function below is reduced to:

    std::chrono::milliseconds
    get () const
    {
        return lastSample_.load();
    }